src/components/TimelineEvent.js

```diff
@@ -3,8 +3,21 @@ import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
 const TimelineEvent = () => {
```

Comment:
I think props should be a parameter in TimelineEvent to call the data for person, status and timeStamp.